nixos/lib/make-system-tarball.sh

```diff
@@ -52,9 +52,10 @@ $extraCommands
 
 mkdir -p $out/tarball
 
-tar cvJf $out/tarball/$fileName.tar.xz * $extraArgs
+rm env-vars
+
+tar --sort=name --mtime='1970-01-01' -cvJf $out/tarball/$fileName.tar.xz * $extraArgs
```

Comment:
Ah, you're right ofc, but on my end it says `nixbld:nixbld` which seems pretty predictable to me, so it's at least better than it was :)